src/ProfileUserInfo/CurrentUser.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import Cookies from "js-cookie";
+import { useLazyQuery, useQuery } from "@apollo/client";
+import styled from "styled-components";
+import { GET_USER_INFO, LOGOUT } from "../GraphqlOperations/queriesContants";
+import SettingsCard from "../HOC/SettingsCard";
+import { Link } from "react-router-dom";
+import { withRouter, RouteComponentProps } from "react-router";
+
+const ProfileInfo = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const CurrentUser = (props: RouteComponentProps) => {
+  const { data } = useQuery(GET_USER_INFO);
+  const [logoutUser] = useLazyQuery(LOGOUT);
+
+  const onLogoutClick = () => {
+    logoutUser({
+      variables: { key: Cookies.get("token") }
+    });
+    Cookies.remove("token");
+    props.history.push("/");
+  };
+
+  const [open, setOpen] = useState(false);
+  if (data.getUserInfo) {
+  }
+  const { name, login, features } = data.getUserInfo;
+
+  const initialValues = {
+    name,
+    login,
+    admin: features.includes("create")
```

Comment:
seeing this string second time. Also, I'm not sure that this is reliable way to check user role